arangod/Graph/BaseOptions.cpp

```diff
@@ -460,6 +460,10 @@ void BaseOptions::injectTestCache(std::unique_ptr<TraverserCache>&& testCache) {
   _cache = std::move(testCache);
 }
 
+arangodb::aql::FixedVarExpressionContext const& BaseOptions::getExpressionCtx() const {
+  return _expressionCtx;
+}
+
```

Comment:
Will those options stay after we're done with the refactoring? Or do we need to move those things out?